test/safe/branch.test.ts

```diff
@@ -59,14 +68,6 @@ describe("Branch Webhook", () => {
 			const ref = encodeURIComponent("heads/TES-123-test-ref");
 			const sha = "test-branch-ref-sha";
 
-			githubNock.post(`/app/installations/1234/access_tokens`)
-				.reply(200, {
-					expires_at: Date.now() + 3600,
-					permissions: {},
-					repositories: {},
-					token: "token"
-				})
```

Comment:
This was wrong. Firstly this was only in a single test, so it was relying that the first test will be executed first and cache the token. 

Secondly it should be Optional, because if token is already cached then we won't request it again.